### DISCUSSION TITLE
Global variable($global variable) use in user script

### DISCUSSION THREAD

**Discussion by pppaw** [foxglove/community#756] [General]
I want to update value in user script by using Global variable($global variable), but it is not taking value. Instead of that it is showing error as, it is declared as an object and you are using it as a value.
declaration:
type GlobalVariables = { someNumericaVar: number };

use
if (rcs > globalVars.someNumericaVar && rcs < 9)

Anyone know, how to use global variable to update value in user script  ?

**Comment by jtbandes** [comment]
Can you please share the whole user script you have written?
